playbooks/provisioner/foreman/pre.yaml

```diff
@@ -0,0 +1,18 @@
+---
+- name: Pre tasks
+  hosts: localhost
+  gather_facts: no
+  tasks:
+    - name: Add candidate hosts to host list
+      add_host:
+        name="{{ item.value.name }}"
```

Comment:
please use:
```yaml
module:
    argument: value
```
Instead of
```yaml
module:
    argument=value
```

And try to keep 4 spaces indentation